tests/workspace/test_workflows.py

```diff
@@ -122,37 +116,44 @@ def test_print_key():
             },
         },
     ]
-    assert json.loads(jobs.get_text_blocks_for_key()) == blocks
+    assert json.loads(jobs.get_text_blocks_for_key(None)) == blocks
 
 
-@patch("workspace.workflows.jobs._get_command_line_args")
-def test_no_target(args):
-    args.return_value = {"key": False}
-    with pytest.raises(ValueError):
-        jobs.parse_args()
+@pytest.mark.parametrize("org", ["opensafely-core", "osc"])
+def test_org_as_target(org):
+    args = jobs.get_command_line_parser().parse_args(f"show --target {org}".split())
+
+    with patch("workspace.workflows.jobs._main") as mock:
```

Comment:
minor nit: I would call this something that  isn't `mock`, to avoid confusion with `unittest.mock`

Reply:
Changed to `mock__main` :)